portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,33 @@
   </head>
   <body>
     <div id="content">
+
+
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <p>This is Eddy's portfolio.</p>
+      <p>Hello! My name is Eddy Nkere and my hometown is Charlottesville Virginia.</p>
+      <p>One thing that I am quite passionate about is the story line of syfy characters in movies and TV-shows.</p>
+      <p> One of them is the Mad Titan and the Black Order</p>
+      <img src="img/thanos2.jpg" width = "500" height= "300"/>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Click here to learn the names of each one of the children of Thanos:</p>
+      <button onclick="addRandomGreeting()">Click</button>
       <div id="greeting-container"></div>
-    </div>
+      <h2>Background</h2>
+      <p>I was born in the Republic Democratic of Congo, and moved to the United State in 2015.
+          I love being active, by exercising and being in conection with nature. I love board games like: chess, checker, and sudoku. They help me improve my critical and analical ways of thinking. 
+          I am passionate about the Computer Science field for the opportunities it provides to inovate and create new technologies.</p>
+          
+       <h2>Projects</h2>
+       <p>I have worked on projects such as:</p>       
+       <p>MineField, Imaginary-Soldier, and Class-Grades</p>
+       <p>I completed class grades using lists and dictionaries in python, with a set of different grades for students the project had to calculate and return the grades average of individual students and the whole class in general.</p>
+       <p>MineField and imaginary Soldiers were completed using java; MineField consisted of a 2 dimentional board field with a specific number of mines desired by user. I desided the game using basic java tools such as Arrays, this was my proper introduction to Java.</p>
+       <h2>Links</h2>
+       <p>Here is a link to my <a href="www.linkedin.com/in/eddy-nk-tothemoon">LinkedIn</a> account.</p>
```

Comment:
If you ever want a link to open in a new tab, you can add the target attribute with value _blank. I'd also suggest including the https:// prefix for the link.

Like this:
`<a href="www.linkedin.com/in/eddy-nk-tothemoon" target="_blank">LinkedIn</a>`

Reply:
I will try to make these modifications